### DISCUSSION TITLE
display each individual value only once

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#5311] [Q&A] [answered]
I'm trying to achieve what was discussed here:
https://statamic.com/forum/3989-only-output-each-unique-value-instance-once

Except my structure loops through an asset fieldtype like this code below (where leather is a taxonomy term fieldtype in the asset blueprint)

```
{{ product_image as="options" }}
	{{ options unique="true" }}
		<p>{{ leather:slug }}</p>
	{{ /options }}
{{ /product_image }}
```

If I display all leather:slug variations I would get something like "red red red blue blue green green green" what I would like is to get "red blue green"

Currently, with unique="true" I only get the first option displayed.

**Comment by edalzell** [comment]
does each image have the same options?

**Reply by fredbouin** [reply]
each image has the same set of possible options but each image has a different option selected.

**Comment by edalzell** [comment]
and you want to list ALL options? Or just the ones that each image has, in one unique list?

**Reply by fredbouin** [reply]
The ones that each image has, in one unique list.

example:
possible taxonomy terms = red, blue, green, yellow
images selected: image-one = red, image-two = red, image-three = yellow.

I would like to display: red, yellow.
not: red red yellow
nor do i want: red blue green yellow

**Comment by edalzell** [comment, marked as answer]
Ya you're in custom modifier/tag territory here.

**Reply by fredbouin** [reply]
ah too bad, ok, from the previous post I thought it might be possible out of the box, thanks for taking the time.
